folio-spring-i18n/src/main/java/org/folio/spring/i18n/config/TranslationConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.spring.i18n.config;
+
+import java.util.Locale;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * An {@link org.springframework.context.annotation.Configuration Configuration} bean to hold the
+ * location of the current translation directory and default fallback locale.
+ */
+@Getter
+@Configuration
+public class TranslationConfiguration {
```

Comment:
I believe this configuration should be integrated into the auto-import functionality provided by Spring Boot. It's been implemented in other submodules and could be beneficial for developers, enabling them to skip importing this configuration into the module configuration:
https://github.com/folio-org/folio-spring-support/blob/master/folio-spring-cql/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

And maybe it make sense to add ComponentScan annotations for service package.

Reply:
Just to make sure I understand, you'd like me to:
- create the auto config resource pointing to `TranslationConfiguration`
- Add `@ComponentScan` to `TranslationService` (pointing just to the service package?)

Reply:
Yes, because without this it could be not registered in application context